src/Database/Attach/File.php

```diff
@@ -397,15 +385,15 @@ public function getContentType()
     /**
      * Get file contents from storage device.
      */
-    public function getContents($fileName = null)
+    public function getContents(?string $fileName = null): string
     {
         return $this->storageCmd('get', $this->getDiskPath($fileName));
     }
 
     /**
      * Returns the public address to access the file.
      */
-    public function getPath($fileName = null)
+    public function getPath(?string $fileName = null): string
```

Comment:
Adding type hints to all of these methods means that we'll need to propagate them to the System\Models\File model as well as any userland extensions of that class.

Reply:
@bennothommo ping

Reply:
What would you prefer? Roll them back or mark it as a breaking change? I'm erring towards marking it as breaking, since it would be better to have full type hinting, and I can't imagine too many people would've extended this model.